lxd/containers.go

```diff
@@ -117,6 +96,52 @@ func containersPost(d *Daemon, r *http.Request) Response {
 	return &asyncResponse{run: run, containers: []string{req.Name}}
 }
 
+func createFromNone(d *Daemon, req *containerPostReq) Response {
+	dpath := shared.VarPath("lxc", req.Name)
+	if !shared.PathExists(dpath) {
+		return InternalError(fmt.Errorf("Container path doesn't exist."))
+	}
```

Comment:
Yeah, let's remove that.

Down the line, we'll have support for pointing to an existing block device in "none" mode so that check wouldn't make much sense then anyway.